src/components/Article.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, text, date }) => (
+  <>
+    <h1>
+      {title}
+    </h1>
+
```

Comment:
```suggestion
    <h1> {title} </h1>
```
Если нет атрибутов, пишем в 1 строку с контентом тега. пробелы между элементами также не нужны.